crnk-validation/src/main/java/io/crnk/validation/ValidationModule.java

```diff
@@ -31,7 +32,7 @@ public String getModuleName() {
 	public void setupModule(ModuleContext context) {
 		context.addExceptionMapper(new ConstraintViolationExceptionMapper(context));
 		context.addExceptionMapper(new ValidationExceptionMapper());
-	}
-
 
+		context.addRepositoryFilter(new ValidationRepositoryFilter());
```

Comment:
looks great, one last minor thing, could add a boolean property to the module whether to enable/disable the filter to allow backward compatibility?

depending on the use cases, the repositories may want to execute the validator by itself. But default true like it is now seems good.